source/documentation/common/upgrade/proc-Upgrading_hypervisor_preserve_gluster_storage.adoc

```diff
@@ -313,8 +313,7 @@ and
 +
 [source,terminal,subs="normal"]
 ----
-# cd /etc/ansible/roles/gluster.ansible/playbooks/hc-ansible-deployment
-ansible-playbook he.yml --extra-vars='@he_gluster_vars.json'
+# cd /etc/ansible/roles/gluster.ansible/playbooks/hc-ansible-deployment ansible-playbook he.yml --extra-vars "@he_gluster_vars.json"
```

Comment:
I don't think this will work. If you want to put the command on the same line, you probably need a ";" separating the "cd" and "ansible-playback" commands.

@sandrobonazzola What do you think?

Reply:
@apinnick indeed, at least a `;` is needed. I would also consider using a separate command line.

Reply:
will this work?
# cd /etc/ansible/roles/gluster.ansible/playbooks/hc-ansible-deployment
# ansible-playbook he.yml --extra-vars "@he_gluster_vars.json"